controllers/suite_test.go

```diff
@@ -74,17 +81,30 @@ func TestAPIs(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	done := make(chan interface{})
-	go func() {
 
+	// log msgs from reconciler to a log file
+	var f io.Writer
+	f, err := os.Create("/tmp/ocs-osd-deployer.log")
```

Comment:
If we decide to go with this implementation, let's add a timestamp to the file name, 
`/tmp/ocs-osd-deployer-YYYYMMDD`

Reply:
- May I know the reason behind it? I don't think we need to have timestamped logs, if need may arise we can rename the file manually